salt/_modules/metalk8s.py

```diff
@@ -4,16 +4,19 @@
 '''
 import logging
 
+from salt.exceptions import CommandExecutionError
```

Comment:
Those changes seem totally unrelated, should be done in a dedicated commit (or dedicated PR).

Or am I wrong and it is related to the renaming?